libs/@local/hashql/eval/src/graph/mod.rs

```diff
@@ -0,0 +1 @@
+pub mod read;
```

Comment:
[Nitpick]
Suggestions on name. `read` is fine, no objections, but in alignment of H-4538, we might want to chose a different name
- `query`
- `select`
- `tail`

Reply:
In that case, I'd gravitate towards `Select` (so that in the future we can have `Update` and `Delete`, normal db terminology). I would do that rename (as it also touches a fair part of the HIR) in a separate PR, is that okay?